assets/component-card.css

```diff
@@ -214,16 +214,10 @@
   }
 }
 
-.card-information quantity-popover volume-pricing {
+.card-information quantity-popover div:not(.quantity__rules) ~ volume-pricing {
   margin-top: 4.2rem;
 }
 
-@media screen and (min-width: 990px) {
-  .card-information quantity-popover volume-pricing {
-    margin-top: 0;
-  }
```

Comment:
<img width="429" alt="Screenshot 2024-04-09 at 2 23 56 PM" src="https://github.com/Shopify/dawn/assets/19962891/25f78c94-f4ab-47e8-bdf3-e26759efaa5a">

Well want to add this back here too